src/components/feilvisning/FeilTabell.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import {Knapp} from "nav-frontend-knapper";
+import {Ingress} from "nav-frontend-typografi";
+import Vis from "../Vis";
+
+interface feilTabellProps {
+  feil : Feil[]
+  visAlleFeil: boolean
+  handleSetVisAlleFeil: (passedValue: boolean) => void
+}
+
+interface Feil {
```

Comment:
Dette interfacet har jeg definert både her og i hovedfila, er det noe jeg kan gjøre for å bare definere det ett sted og så bruke det begge plasser?

Reply:
Vi har samlet det i sporenstreksTypes, men da bør du nok bruke et litt mindre generisk navn.